content/nl/about/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Over
+description: Wat is het "Moodle Gereedschapsgids"?
+---
+
+_Moodle Gereedschapsgids_ is een referentiekaart voor het gebruik van [Moodle][moodle] versie 3 en later, ontworpen voor leerkrachten, opleiders en trainers. Het geeft informatie over de belangrijkste Moodle-tools en welke tool gebruikt moet worden in overeenstemming met het pedagogische doel dat bereikt moet worden. Het vermeldt ook de taxonomische niveaus van Bloom die bereikt kunnen worden bij het gebruik van de verschillende tools.
+
+Een idee van [Joyce Seitzinger](https://twitter.com/catspyjamasnz) van [Eastern Institute of Technology](https://www.eit.ac.nz/) (NZ). In mei 2020 aangepast voor Moodle 3.x door [Nicolas Martignoni][nm], gebaseerd op het werk van versi 2 door [Gavin Henrick](https://twitter.com/ghenrick).
```

Comment:
```suggestion
Een idee van [Joyce Seitzinger](https://twitter.com/catspyjamasnz) van [Eastern Institute of Technology](https://www.eit.ac.nz/) (NZ). In mei 2020 aangepast voor Moodle 3.x door [Nicolas Martignoni][nm], INSERT HERE CREDITS FOR DUTCH TRANSLATION.
```
See e.g. [Spanish version](https://github.com/martignoni/moodle-tool-guide/blob/master/content/es/about/index.md), line 8.

Reply:
Hi Nicolas,
I am a little bit new in this all but if I am correct I changed the file as requested. Please let me know if not.
Gemma